netdb.c

```diff
@@ -80,11 +93,20 @@ struct hostent *RETRACE_IMPLEMENTATION(gethostbyaddr)(const void *addr, socklen_
 
 	retrace_log_and_redirect_before(&event_info);
 
-	hent = real_gethostbyaddr(addr, len, type);
-	if (hent) {
-		event_info.return_value_type = PARAMETER_TYPE_STRUCT_HOSTEN;
-	} else
-		event_info.logging_level |= RTR_LOG_LEVEL_ERR;
+	if (rtr_get_net_fuzzing(event_info.function_name, &err)) {
```

Comment:
Why not `rtr_get_net_fuzzing(NET_FUNC_GETHOSTBYADDR, &err)` so `rtr_get_net_fuzzing` wouldn't need to do any of the `strcmp` loop (discussed above). Also could `rtr_get_net_fuzzing` etc. go into new files `netfuzz.c` and `netfuzz.h`?